### ISSUE TITLE
RAPID: should all namespaces be default?

### ISSUE THREAD

**Issue by mikepizzo** [oasis-open/odata-rapid#211]
Odata 4.01 introduced the concept of default namespaces, so that clients don't have to specify the namespace in URLs (and we defined a set of precedence rules for resolving ambiguity between, i.e., members and bound operations).

Also, services aren't required to namespace-qualify type references from the default namespace in the response.

How should RAPID deal with namespaces?

Proposal: Clients should only have to specify the namespace in URLs to disambiguate.
What about type references in the request/response body? (i.e., @odata.type)?

**Comment by chrisspre** [comment]
https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Core.V1.xml#L371

**Comment by ralfhandl** [comment]
Yes, all namespaces are default namespaces. It's best practice set by MS Graph 😄

**Comment by chrisspre** [comment]
The implementation of this has to wait a little bit. The library used to generate CSDL [Microsoft.OData.Edm](https://www.nuget.org/packages/Microsoft.OData.Edm/) currently doesn't support annotating Includes which is necesarry to mark an include as default (see [DefaultNamespace](https://github.com/oasis-tcs/odata-vocabularies/blob/master/vocabularies/Org.OData.Core.V1.xml#L371) )

**Comment by mikepizzo** [comment]
There are really two things:
1) Does the service support omitting certain namespaces when referencing actions/functions/types in the URL
2) Does the service *advertise* that it supports omitting certain namespaces when referencing actions/functions/types in the URL

I think we should say that RAPID services don't require namespace qualification in the URL (i.e., in our samples, etc.).

Separately, we should update the RSDL->CSDL conversion tool to annotate the CSDL appropriately. This is dependent upon a change to Microsoft.OData.Edm (note that we already have a PR for this change in code review).

**Comment by mikepizzo** [comment]
Mike to update documentation.
